src/main/kotlin/strdcoders/mss/schedule/StorageTriggerScheduler.kt

```diff
@@ -0,0 +1,31 @@
+package strdcoders.mss.schedule
+
+import org.slf4j.LoggerFactory
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Service
+import strdcoders.mss.services.PartitionMonitorService
+import strdcoders.mss.services.RuleFlowRunner
+import strdcoders.mss.utils.StorageUtils
+import java.util.concurrent.TimeUnit
+
+@Service
+class StorageTriggerScheduler(
+    private val partitionMonitorService: PartitionMonitorService,
+    private val storageUtils: StorageUtils,
+    private val ruleFlowRunner: RuleFlowRunner,
+) {
+
+    private val log = LoggerFactory.getLogger(javaClass)
+
+    @Scheduled(initialDelay = 2, fixedRate = 1, timeUnit = TimeUnit.SECONDS)
```

Comment:
Is that just a temporary values? Use configurations / env variable?

Reply:
https://github.com/STRDCoders/mss/pull/2/commits/76c5871501eb636211210d1b0cf5ef31a2e41c47